app/assets/javascripts/admin/order_cycles/controllers/order_cycles_controller.js.coffee

```diff
@@ -6,6 +6,13 @@ angular.module("admin.orderCycles").controller "OrderCyclesCtrl", ($scope, $q, C
   $scope.involvingFilter = 0
   $scope.scheduleFilter = 0
 
+  $scope.resetSelectFilters = ->
```

Comment:
We should add a unit test for this new function in [spec/javascripts/unit/admin/order_cycles/controllers/order_cycles_controller_spec.js.coffee](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/spec/javascripts/unit/admin/order_cycles/controllers/order_cycles_controller_spec.js.coffee)